### DISCUSSION TITLE
Bounding Box table

### DISCUSSION THREAD

**Discussion by Akhilesh-pandey1** [deepdoctection/deepdoctection#287] [Q&A]
Through deepdoctection, I did table detection and got the bounding box. So I want to know, How to convert this bounding box for the table_region or table_area = [] for camelot  ?? I tried a lot, not get any success. So if you tell me to the bounding box of the deepdoctection, for the camelot ? 
Thank you so much.

**Comment by JaMe76** [comment]
I am sorry, but I don't know anything about Camelot. 

However, I will convert this issue into a discussion, so somebody might know an answer to to your question.
